Analyze case with vlinder.ipynb

```diff
@@ -5,23 +5,42 @@
    "metadata": {},
    "source": [
     "## Model: the Responsible Business Simulator (tRBS)\n",
-    "In this notebook the core (calculation) steps of the tRBS model are illustrated.\n",
+    "In this notebook the core (calculation) steps of the tRBS model are illustrated. We import this model via Python package vlinder.\n",
     "\n",
     "### `TheResponsibleBusinessSimulator` class\n",
-    "In order to work with the Responsible Business Simulator model, we will work with the ```TheResponsibleBusinessSimulator``` class. This class can be found in **core/trbs.py**. This class takes three input values: (i) the folder where data of the case can be found, (ii) the format of the data (options: xlsx, json, csv) and (iii) the name of the case (options: Beerwiser)."
+    "In order to work with the Responsible Business Simulator model, we will work with the ```TheResponsibleBusinessSimulator``` class. This class can be found in Python package vlinder, in **src/vlinder/trbs.py**. This class takes three input values: (i) the folder where data of the case can be found, (ii) the format of the data (options: xlsx, json, csv) and (iii) the name of the case (options: beerwiser, DSM, IZZ and refugee)."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install required dependencies for package vlinder\n",
+    "!pip install pandas\n",
+    "!pip install numpy\n",
+    "!pip install openpyxl\n",
+    "!pip install matplotlib\n",
+    "!pip install xlsxwriter\n",
+    "!pip install vlinder --upgrade"
```

Comment:
waarom hier upgrade en bij andere packages niet? kunnen we dit omzeilen?

Reply:
Omdat bij vlinder pakt hij niet de latest versie zonder --upgrade, als je al een oudere versie van vlinder hebt geïnstalleerd. Met --upgrade pakt hij wel altijd de latest versie van vlinder.